tests/phpunit/tests/theme/wpAddGlobalStylesForBlocks.php

```diff
@@ -73,6 +73,29 @@ public function test_third_party_blocks_inline_styles_get_registered_to_global_s
 		);
 	}
 
+	/**
+	 * @ticket 59595
+	 */
+	public function test_check_cache_is_set() {
```

Comment:
I think the name could more clearly describe what cache is being tested.

```suggestion
	public function test_styles_for_blocks_cache_is_set() {
```

Reply:
Handled in [d0b243c](https://github.com/WordPress/wordpress-develop/pull/6392/commits/d0b243cdd828e1ae3d03ad206918574de9fab94b).